djstripe/event_handlers.py

```diff
@@ -205,6 +207,8 @@ def other_object_webhook_handler(event):
         target_cls = models.Dispute
     else:
         target_cls = {
+            "account": models.Account,
+            # TODO: Add an Application model to handle account.application.auth or de-auth events
```

Comment:
What happens if you receive such an event right now? Will it crash?

Reply:
Outdated